packages/kununu-theme-v2/scss/effects/core.scss

```diff
@@ -0,0 +1,210 @@
+  padding: $grid-gutter-width;
+}
+
+.padding-half {
+  padding: $grid-gutter-width / 2;
+}
+
+.padding-quarter {
+  padding: $grid-gutter-width / 4;
+}
+
+.padding-top {
+  padding-top: $grid-gutter-width;
+}
+
+.padding-bottom {
+  padding-bottom: $grid-gutter-width;
+}
+
+.padding-bottom-double {
+  padding-bottom: $grid-gutter-width * 2;
+}
+
+.padding-bottom-quadruple {
+  padding-bottom: $grid-gutter-width * 4;
+}
+
+.padding-top-half {
+  padding-top: $grid-gutter-width / 2;
+}
+
+.padding-right-half {
+  padding-right: $grid-gutter-width / 2;
+}
+
+.padding-bottom-half {
+  padding-bottom: $grid-gutter-width / 2;
+}
+
+.padding-top-third {
+  padding-top: $grid-gutter-width / 3;
+}
+
+.padding-bottom-third {
+  padding-bottom: $grid-gutter-width / 3;
+}
+
+.padding-top-fourth {
+  padding-top: $grid-gutter-width / 4;
+}
+
+.padding-bottom-fourth {
+  padding-bottom: $grid-gutter-width / 4;
+}
+
+.padding-right {
+  padding-right: $grid-gutter-width;
+}
+
+.padding-left {
+  padding-left: $grid-gutter-width;
+}
+
+.padding-left-half {
+  padding-left: $grid-gutter-width / 2;
+}
+
+.margin {
+  margin: $grid-gutter-width;
+}
+
+.margin-third {
+  margin: $grid-gutter-width / 3;
+}
+
+.margin-top {
+  margin-top: $grid-gutter-width;
+}
+
+.margin-top-half {
+  margin-top: $grid-gutter-width / 2;
+}
+
+.margin-top-third {
+  margin-top: $grid-gutter-width / 3;
+}
+
+.margin-left {
+  margin-left: $grid-gutter-width;
+}
+
+.margin-left-half {
+  margin-left: $grid-gutter-width / 2;
+}
+
+.margin-left-third {
+  margin-left: $grid-gutter-width / 3;
+}
+
+.margin-right {
+  margin-right: $grid-gutter-width;
+}
+
+.margin-right-half {
+  margin-right: $grid-gutter-width / 2;
+}
+
+.margin-right-third {
+  margin-right: $grid-gutter-width / 3;
+}
+.margin-right-quarter {
+  margin-right: $grid-gutter-width / 4;
+}
+
+.margin-bottom {
+  margin-bottom: $grid-gutter-width;
+}
+
+.margin-bottom-double {
+  margin-bottom: $grid-gutter-width * 2;
+}
+
+.margin-bottom-half {
+  margin-bottom: $grid-gutter-width / 2;
+}
+
+.margin-bottom-third {
+  margin-bottom: $grid-gutter-width / 3;
+}
+
+.margin-bottom-negative-double {
+  margin-bottom: -$grid-gutter-width * 2;
+}
+
+.margin-vertical {
+  @extend .margin-top;
+  @extend .margin-bottom;
+}
+
+.margin-vertical-half {
+  @extend .margin-top-half;
+  @extend .margin-bottom-half;
+}
+
+.margin-vertical-third {
+  @extend .margin-top-third;
+  @extend .margin-bottom-third;
+}
+
+.margin-horizontal-half {
+  @extend .margin-left-half;
+  @extend .margin-right-half;
+}
+
+.inline-block {
+  display: inline-block;
+}
+
+.block {
+  display: block;
+}
+
+.table-cell {
+  display: table-cell;
+}
+
+.fakelink {
+  cursor: pointer;
+}
+
+.pointer-disabled {
+  pointer-events: none;
+}
+
+.bullet-after:after {
+  content: '\2022';
+  padding: 0 8px;
+
+  @media (max-width: $screen-sm-min) {
+    display: block;
+    content: '';
+  }
+}
+
+.bullet-before:before {
+  content: '\2022';
+  padding: 0 8px;
+}
```

Comment:
I don't expect we will use them very often. Should we maybe remove these from the start?

Reply:
Well pete added these for the profile page. They love to use bullets! I would leave this for now. I think it will be useful when we implement review tab.